low-level/framework/sspl_reinit

```diff
@@ -51,7 +51,7 @@ usermod -a -G disk sspl-ll
 usermod -a -G disk sspl-ll
 
 # Update the sudoers file with the sspl-ll user and available commands
-SUDO_LINE="sspl-ll	ALL = NOPASSWD: /usr/sbin/smartctl, /usr/sbin/mdadm, /usr/bin/mount, /usr/bin/umount, /usr/sbin/swapon, /usr/sbin/swapoff, /usr/sbin/hdparm, /usr/bin/systemctl, /usr/sbin/wbcli, /usr/bin/ipmitool, /usr/bin/systemd-detect-virt, /bin/salt-call, /bin/tee, /usr/bin/facter, /usr/bin/provisioner"
+SUDO_LINE="sspl-ll	ALL = NOPASSWD: /usr/sbin/smartctl, /usr/sbin/mdadm, /usr/bin/mount, /usr/bin/umount, /usr/sbin/swapon, /usr/sbin/swapoff, /usr/sbin/hdparm, /usr/bin/systemctl, /usr/sbin/wbcli, /usr/bin/ipmitool, /usr/bin/systemd-detect-virt, /bin/salt-call, /bin/tee, /usr/bin/facter, /usr/bin/provisioner, /usr/bin/consul"
```

Comment:
why we need here now ?

Reply:
We had seen pam_auth permission error logged in systemctl when sspl-ll user executes tool/commands in shell from .py scripts. That's why I added this command to sspl sudoers.
I will do a test and confirm once.

Reply:
yeah add only if we know its needed for sure and dont go by observations for other binaries